content/ai/concepts/neural-networks/terms/gaussian-activation-function/gaussian-activation-function.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Gaussian Activation Function'
+Description: 'The Gaussian activation function takes the input and transforms it into a curve that looks like a bell, where the resulting values are between 0 and 1.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+A **gaussian activation function** is a mathematical function also known as the normal distribution or bell curve function. It is a fundamental concept in statistics and probability theory often used in AI and neural networks as a probability distribution function to model the output of a neuron. 
+
+## Gaussian Function
+
+Characterized by its bell-shaped graph, where most data points are near the middle (average), and the likelihood of getting values decreases as you move away from the average. This curve is symmetric, meaning that the probabilities of getting values above the
+average are the same as getting values below the average.
+
+![1D Gaussian function](<../../../../../../media/1D Gaussian Function.png>)
+
+![2D Gaussian function](<../../../../../../media/2D Gaussian function.png>)
```

Comment:
This is not the right way to link your medit. It should be like this: https://raw.githubusercontent.com/Codecademy/docs/main/media/css-display-float-example.png

Also rename your image files instead of space use `-`.